test/pages/IsTheSaleSubjectToConditionsPageSpec.scala

```diff
@@ -28,4 +31,18 @@ class IsTheSaleSubjectToConditionsPageSpec extends PageBehaviours {
 
     beRemovable[Boolean](IsTheSaleSubjectToConditionsPage)
   }
+  "cleanup" - {
+    "should reset DescribeTheConditionsPage" - {
+      "when IsTheSaleSubjectToConditionsPage is changed to No" in {
+
+        val emptyUserAnswers = UserAnswers("id", DraftId(1))
+        val ua               = emptyUserAnswers
+          .set(DescribeTheConditionsPage, "conditions")
+          .get
+
+        IsTheSaleSubjectToConditionsPage.cleanup(Some(false), ua) mustBe
+          Success(emptyUserAnswers)
+      }
+    }
```

Comment:
We want to be testing all scenarios here, namely what happens when the cleanup is called with a None, and Some(true).